src/index.md

```diff
@@ -5,7 +5,7 @@
 :::
 
 * [Introduction](intro.html)
-  * [Release Notes](intro.html#release-notes)
+  * [Maintenance](intro.html#maintenance)
```

Comment:
Don't think this should be here anymore.

Reply:
Doh.. Fixed, and fixed again. I wrote a script to generate the index automatically at make time, that should avoid these issues in the future. A few corner cases that needs to be fixed before I push that though.